extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -116,15 +116,85 @@ export class WindowPortEmulator {
  * ampdoc and Bob is the viewer.
  */
 export class Messaging {
+  /**
+   * Performs a handshake and initializes messaging.
```

Comment:
So, if we're not changing the viewer params at this time, let's include them here. E.g. "To use this mode the document's viewer params must include cap=handshakepoll"...

Reply:
Done

Reply:
Hmm. Not seeing it. Did you push new changes?